src/lib/components/forms/form/form-entry/terra-form-entry.base.ts

```diff
@@ -142,4 +147,15 @@ export class TerraFormEntryBase implements OnChanges, OnDestroy {
     private _transformInputPropertyName(propertyName: string): string {
         return 'input' + propertyName.charAt(0).toUpperCase() + propertyName.substr(1);
     }
+
+    /**
+     * Removes the prefix 'input' from the given propertyName and lowercases the succeeding character.
+     * @param propertyName
+     */
+    private _removeInputPrefix(propertyName: string): string {
+        if (!propertyName.startsWith('input')) {
+            return propertyName;
+        }
+        return propertyName.charAt(5).toLowerCase() + propertyName.substr(6);
```

Comment:
optional aber wegen der magic number
```suggestion
        return propertyName.charAt('input'.length).toLowerCase() + propertyName.substr(6);
```

Reply:
ich habe mich bewusst dagegen entschieden. Ich hätte es wenn nämlich ganzheitlich (mit separater Konstante, etc..) gelöst und dafür auch andere Stellen anpassen müssen. Das wollte ich vermeiden.